geode-redis/src/distributedTest/java/org/apache/geode/redis/GeodeRedisServerStartupDUnitTest.java

```diff
@@ -219,14 +219,14 @@ public void startupFailsGivenIllegalPort() {
         .withProperty(REDIS_PORT, "-1")
         .withProperty(REDIS_BIND_ADDRESS, "localhost")
         .withProperty(REDIS_ENABLED, "true"))).hasRootCauseMessage(
-            "Could not set \"redis-port\" to \"-1\" because its value can not be less than \"0\".");
+            "Could not set \"compatible-with-redis-port\" to \"-1\" because its value can not be less than \"0\".");
   }
 
   @Test
   public void startupFailsGivenPortAlreadyInUse() throws Exception {
     int port = AvailablePortHelper.getRandomAvailableTCPPort();
 
-    addIgnoredException("Could not start Redis Server");
+    addIgnoredException("Could not start Server compatible with Redis");
```

Comment:
So silly thing, but it looks like this DUnit test is failing because it is case sensitive and the s in server should be lowercase.

Reply:
oh yeah, I changed the exception in a previous commit but forgot to change it here. thanks!